route-rs-runtime/src/lib.rs

```diff
@@ -27,4 +27,4 @@ pub mod link;
 pub mod pipeline;
 
 /// Utility module
-mod utils;
+pub mod utils;
```

Comment:
note: I intended for these utilities (namely test utilities) were supposed to be private to the runtime, but it makes sense to export them so users can run tests on their router graphs.

Reply:
Yeah..we will go over all of our ways to run a router this next meetup and refactor it into one coherent tool